.env.example

```
@@ -40,4 +40,8 @@ API_URL=
 DOMAIN=
 
 CLIENT_SUCCESS_URL=
-SWAGGER_URL=
\ No newline at end of file
+SWAGGER_URL=
+
+GRAFANA_USER=
+GRAFANA_PASSWORD=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GRAFANA_PASSWORD key should go before the GRAFANA_USER key

(UnorderedKey)